### PR TITLE
Add HW7 Task2

### PR DESCRIPTION
**Pull request by Halyna2023**

### CODE REVIEW

File: src/ua/olifirenko/hw7/Task2.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+//        в) метод, вычисляющий векторное произведение с другим вектором
+//        г) метод, вычисляющий косинус угла между векторами: косинус угла между векторами
+//        равен скалярному произведению векторов, деленному на произведение модулей (длин)
+//        векторов;
+//        д) методы для суммы и разности векторов
+//        е) статический метод, который принимает целое число N, и возвращает массив случайных
+//        векторов размером N.
+//        *Если метод возвращает вектор, то он должен возвращать новый объект, а не менять
+//        базовый
+public class Task2 {
```
Comment (ArtemKharin) [inline src/ua/olifirenko/hw7/Task2.java:17]: need to rename class to Vector

**Comment by ArtemKharin** [comment]
but where is .gitignore?

File: src/ua/olifirenko/hw7/Task2.java (hunk 1 above)
Comment (Halyna2023) [reply src/ua/olifirenko/hw7/Task2.java:17]: Ok. I renamed

**Comment by Halyna2023** [comment]
я  .gitignore добавила в main

**Comment by Halyna2023** [comment]
необходимо было в hw7?

**Pull request closed without merging**